weave/trace_server/calls_query_builder.py

```diff
@@ -347,13 +353,34 @@ def set_hardcoded_filter(self, filter: HardCodedFilter) -> "CallsQuery":
         return self
 
     def add_order(self, field: str, direction: str) -> "CallsQuery":
+        """Add order."""
         direction = direction.upper()
         if direction not in ("ASC", "DESC"):
             raise ValueError(f"Direction {direction} is not allowed")
         direction = cast(Literal["ASC", "DESC"], direction)
-        self.order_fields.append(
-            OrderField(field=get_field_by_name(field), direction=direction)
-        )
+        # Support special handling for the status field which is computed from exception and ended_at
+        if field == "status":
```

Comment:
I think its going to be worth it to add a more extendable framework here, @zbirenbaum is  thinking about sorting by `latency` which will run into a very similar path. 

We probably want at least a const somewhere with the special fields